src/components/Restaurant/Restaurant.jsx

```diff
@@ -1,23 +1,20 @@
 import { useState } from "react";
+import RestaurantButton from "../RestaurantButton/RestaurantButton";
 
 function Restaurant() {
   const [pizzas, setPizzas] = useState(0);
 
-  function buttonAdd() {
+  const OrderOne = () => {
     setPizzas(pizzas + 1);
-  }
+  };
 
   return (
     <section className="restaurant">
       <h3 className="restaurant__heading">Restaurant Orders</h3>
       <ul className="restaurant__list">
-        <li className="restaurant__item">
-          Pizzas: {pizzas}{" "}
-          <button className="button restaurant__button" onClick={buttonAdd}>
-            Add
-          </button>
-        </li>
+        <li className="restaurant__item">Pizzas: {pizzas} </li>
       </ul>
+      <RestaurantButton OrderOne={OrderOne} />
```

Comment:
Looking good! I've got one comment. In JavaScript attributes and functions, we normally start the name with lowercase letters. We only begin class and React compant names with uppercase letters. It's not complusary but it's a useful convention.